packages/client/hooks/useForm.tsx

```diff
@@ -91,7 +91,7 @@ const useForm = (fieldInputDict: FieldInputDict, deps: any[] = []) => {
             }
           }
           return obj
-        }, {} as FieldState),
+        }, {} as FieldState<T>),
```

Comment:
this little guy means the return value has the same keys as the input value. 
i hate the extra PR bloat, but i hit a runtime error so decided to add a little extra type safety.